drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -470,12 +470,12 @@ static int _set(netdev_t *netdev, netopt_t opt, const void *val, size_t len)
     switch (opt) {
         case NETOPT_ADDRESS:
             assert(len <= sizeof(uint16_t));
-            at86rf2xx_set_addr_short(dev, *((const uint16_t *)val));
+            at86rf2xx_set_addr_short(dev, val);
```

Comment:
Here the order needs to be fixed for network byte-order though.

Reply:
Above you said that get()/set() is using network byte order for addresses, right?

Reply:
Ok, for the short address I truly don't care, as it has no semantics to it.